cmd/humioctl/alerts_install.go

```diff
@@ -27,7 +27,7 @@ import (
 
 func newAlertsInstallCmd() *cobra.Command {
 	var (
-		filePath, url, name string
+		filePath, url, name, runAsUserID string
```

Comment:
Is `runAsUserID` optional? It should be.

Reply:
It is optional in the sense that the default value is (""/string null value) if not specified.